src/content/roadmap/user-experience/index.md

```diff
@@ -0,0 +1,28 @@
+
+Read on to discover some of the changes that will be implemented to allow the barriers-to-entry to be broken down:
+
+## Beyond seed phrases {#no-more-seed-phrases}
+
+Ethereum accounts are protected by a pair of keys that are used to identify accounts (public key) and sign messages (private key). A private key works like a master password that allows complete access to an Ethereum account - allowing funds to be moved. This is not a normal mode of operation for people more familiar with banks and web2 companies who manage accounts on a users behalf after retrieving and storing their personal data. In order for Ethereum to reach mass adoption without relying on centralized third-parties, there must be a very easy, frictionless way for a user to custody their own assets and keep control of their own data, but without having to understand public-private key cryptography and key management.
+
+The solution to this is using smart contract wallets to interact with Ethereum. This creates powerful new opportunities for fraud detection and defense, upgradeability and ways for accounts to be protected if the keys are lost or stolen. Although smart contract wallets exist today, they are awkward to build because they are not very well supported by the Ethereum protocol. Adding the necessary support will lead to great security and useability benefits for ordinary users. This additional support is known as **account abstraction**.
+
+<ButtonLink variant="outline-color" to="/roadmap/account-abstraction/">Read about account abstraction</ButtonLink>
+
+## Nodes for everyone
+
+Users that run their own nodes do not have to trust third parties to provide them truthful data, and they can interact quickly, privately and permissionlessly with the Ethereum blockchain. However, right now running a node requires some technical knowledge and substantial disk space, meaning many people choose to trust intermediaries instead. There are several upgrades that will make running nodes far easier and far less resource intensive. The way data is stored will be changed to use a more space-efficient structure known as a **Verkle Tree**. Also, with [statelessness](/roadmap/statelessness) or [data expiry](/roadmap/statelessness/#data-expiry), Ethereum nodes will not need to store their own copy of the entire Ethereum state data, drastically reducing hard disk space requirements. Light nodes will offer many of the benefits of running a full node but able to run easily on mobile phones or inside simple browser apps.
```

Comment:
```suggestion
Users running nodes do not have to trust third parties to provide them with data, and they can interact quickly, privately, and permissionlessly with the Ethereum blockchain. However, running a node right now requires technical knowledge and substantial disk space, meaning many people must trust intermediaries instead. There are several upgrades that will make running nodes far easier and far less resource intensive. The way data is stored will be changed to use a more space-efficient structure known as a **Verkle Tree**. Also, with [statelessness](/roadmap/statelessness) or [data expiry](/roadmap/statelessness/#data-expiry), Ethereum nodes will not need to store a copy of the entire Ethereum state data, drastically reducing hard disk space requirements. Light nodes will offer many benefits of running a full node but can run easily on mobile phones or inside simple browser apps.
```

or maybe making this two paragraphs.

```suggestion
Users running nodes do not have to trust third parties to provide them with data, and they can interact quickly, privately, and permissionlessly with the Ethereum blockchain. However, running a node right now requires technical knowledge and substantial disk space, meaning many people must trust intermediaries instead. 

There are several upgrades that will make running nodes far easier and far less resource intensive. The way data is stored will be changed to use a more space-efficient structure known as a **Verkle Tree**. Also, with [statelessness](/roadmap/statelessness) or [data expiry](/roadmap/statelessness/#data-expiry), Ethereum nodes will not need to store a copy of the entire Ethereum state data, drastically reducing hard disk space requirements. Light nodes will offer many benefits of running a full node but can run easily on mobile phones or inside simple browser apps.
```

Reply:
I also think we should focus first on light nodes and second on the technology that enables them.